src/test/resources/test.properties.template

```diff
@@ -15,5 +15,8 @@ s3.endpoint=http[s]://<ecshost>[:9020|:9021]
 ## Uncomment to enable vhost-style requests (including the namespace)
 #enableVhost=true
 
+## Uncomment to disable smart client
+#isSmartClient=false
```

Comment:
Default should be true

Reply:
By default it's commented, uncommenting it will disable smart client, so default is false.